src/NServiceBus.Testing/TestableMessageHandlerContext.cs

```diff
@@ -36,13 +43,22 @@ public TestableMessageHandlerContext(IMessageCreator messageCreator)
 
         public SynchronizedStorageSession SynchronizedStorageSession { get; }
 
-        public bool HandlerInvocationAborted { get; private set; }
-
-        public bool HandleCurrentMessageLaterWasCalled { get; private set; }
-
         public Task Send(object message, SendOptions options)
         {
+            var headers = options.GetHeaders();
+
+            if (headers.ContainsKey("NServiceBus.IsSagaTimeoutMessage"))
```

Comment:
I wish we could move those saga-only related things closer to the saga and out of the testable context, but I can't come up with a solution.